packages/components/organisms/f-checkout/src/store/tests/__snapshots__/checkout.module.test.js.snap

```diff
@@ -3,6 +3,7 @@
 exports[`CheckoutModule mutations :: UPDATE_STATE :: should update state with delivery response. 1`] = `
 Object {
   "address": Object {
+    "administrativeArea": undefined,
```

Comment:
Why do we instantiate something as `undefined`?

Reply:
I think because it doesn't exist so the snapshot needs to do something with it. The administrativeArea will only exist if the user inputs it.